apps/src/templates/census2017/CensusForm.jsx

```diff
@@ -280,9 +280,7 @@ class CensusForm extends Component {
                {i18n.censusRequiredSchool()}
              </div>
            )}
-          <ProtectedStatefulDiv
-            ref="schoolInfo"
-          />
```

Comment:
Was anything rendering into this `ProtectedStatefulDiv`?  Does that code need to be removed?

Reply:
This div is the old school multi-dropdown partial. I removed it because this new component completely replaces it. I also removed the reference to the partial  from `yourschool.haml` and the imports for the old dropdowns from `yourschool.js`.  Thanks for the reminder!